src/components/ClubCard.vue

```diff
@@ -13,6 +13,20 @@
           }}
         </div>
       </v-card-text>
+      <v-card-actions>
```

Comment:
It may cause enspire to be less important (maybe we should provide a similar tool for clubs to fully shift their own clubpages to enspire).
Maybe useful and more acceptable for clubs in beginning periods of enspire.

Reply:
This is useful at all time IMO—We can't count on users to visit Enspire every day. After all, people are getting less used to browsing websites. Wechat subscriptions and website links are musts, and we should maintain a "minimal yet indispensible" need of Enspire.